src/components/QandA/QandA.js

```diff
@@ -47,7 +47,7 @@ var QandA = () => {
   }, [])
 
   return(
-    <div className="qa">
+    <div className="qa" onClick={props.onClick}>
```

Comment:
careful calling your custom click handlers the same name as a native event in the browser. the idea is to say "on click, this function is run" the "this function" part is "handleClick" in the most generic sense of the concept. ("on click, handle the click")